setup/helpers/abs_helper.rb

```diff
@@ -0,0 +1,284 @@
+require 'net/http'
+require 'timeout'
+require 'json'
+require 'yaml'
+
+module AbsHelper
+
+  # TODO: replace when ready
+  #ABS_BASE_URL = 'https://cinext-abs.delivery.puppetlabs.net/api/v2'
+  ABS_BASE_URL = 'https://cinext-abs-test.delivery.puppetlabs.net/api/v2'
+
+  ABS_AWS_PLATFORM = 'el-7-x86_64'
+  ABS_AWS_IMAGE_ID = 'ami-1b17242b'
+  ABS_AWS_SIZE = 'c4.2xlarge'
+  ABS_AWS_MOM_SIZE = 'c4.2xlarge'
+  ABS_AWS_METRICS_SIZE = 'c4.2xlarge'
+  ABS_AWS_REGION = 'us-west-2'
+
+  ABS_BEAKER_TYPE = 'centos-7-x86-64-west'
+  ABS_BEAKER_ENGINE = 'aws'
+
+  # TODO: replace when ready
+  # ABS_AWS_REAP_TIME = '86400'
+  ABS_AWS_REAP_TIME = '1200'
+
+  def abs_get_base_url
+    ENV['ABS_BASE_URL'] = ABS_BASE_URL unless ENV['ABS_BASE_URL']
```

Comment:
Ok, so after taking a look at the entire file, I think there's a simpler way to do this without needing all of the abs_get_* methods. I'm leaving the additional comments I already made because I think they're useful to learn from, but you don't need to do both things.
Typically what I would do with Ruby here is pretty simple. You can create a constructor that gets all of the final values for you, then just access them as properties. So... like this:
```
def initialize
  @abs_base_url = ENV['ABS_BASE_URL'] ? ENV['ABS_BAS_URL'] : ABS_BASE_URL
  ...
end
```

Reply:
I've updated abs_helper to use this style, thanks!